### ISSUE TITLE
Mermaid: improve link discoverability

### ISSUE THREAD

**Issue by ofek** [squidfunk/mkdocs-material#5619]
### Description

Currently, there is no way to tell what parts of diagrams can be clicked other than hovering over to see if the icon changes

### Related links

Here is an example https://ofek.dev/pyapp/latest/runtime/

### Use Cases

It is very common to have parts of diagrams link to other places in your documentation for elaboration

### Before submitting

- [X] I have read and followed the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
- [X] I have verified that [my idea is a change request and not a bug report](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#its-not-a-bug-its-a-feature).
- [X] I have ensured that, to the best of my knowledge, [my idea will benefit the entire community](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#benefit-for-the-community).
- [X] I have included relevant links to [the documentation](https://squidfunk.github.io/mkdocs-material), related [issues](https://github.com/squidfunk/mkdocs-material/issues), and [discussions](https://github.com/squidfunk/mkdocs-material/discussions) to underline the need for my idea.

**Comment by squidfunk** [comment]
Thanks for suggesting. Does Mermaid.js itself style those links in another way? If not, what's your suggestion?

**Comment by ofek** [comment]
I think the only built-in styling is the [tooltip](https://mermaid.js.org/syntax/flowchart.html#interaction) (which doesn't work as far as I can tell).

Your idea is interesting, that might work. I was also thinking maybe the text could have the standard appearance of a hyperlink. Or maybe a small icon similar to https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-status

Anything is fine, I trust your design decisions 🙂

**Comment by ofek** [comment]
Have you thought about what design you want here?

**Comment by squidfunk** [comment]
Sorry, I currently have no bandwidth to address this topic, since we're investing our time to work on other features like the new search and friends . I'll come back to this later, probably at the end of this year.

I'm sorry for the inconvenience this is causing you. You know how it is, I can't split myself 😅

**Comment by kamilkrzyskow** [comment]
In https://github.com/squidfunk/mkdocs-material/discussions/6778#discussioncomment-8751560 I made a proposal to have an option to add custom styles for the mermaid blocks:
```yml
extra:
  extra_shadow_root_css:
    - assets/stylesheets/mermaid.css # I don't know if relative paths are supported in the shadow root
    - https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css
```
this would allow to apply custom styling to the elements. In this case `xlink:href="../config/#full-isolation"` or `clickable` in `class="node default clickable flowchart-label"` could be used to select the elements and style them accordingly.

**Comment by squidfunk** [comment]
Fixed in 05661a156. We've decided to go with underlines for now, since we're also going to add them in the text in the near future when we do a face lift of the current design.

<img width="659" alt="Bildschirm­foto 2024-05-03 um 19 29 39" src="https://github.com/squidfunk/mkdocs-material/assets/932156/0311f063-b426-4b83-9bff-937e03f7491b">

**Comment by ofek** [comment]
Oh my goodness, that's awesome, thank you!

**Comment by ofek** [comment]
Feel free to close this, but out of curiosity what is the facelift you have planned?

**Comment by squidfunk** [comment]
Modernize the design that is from currently Material Design from 2017. Some users criticized that we're implementing Material Design v1 and not v3, so we're aiming at incorporating the best parts of v3.

**Comment by squidfunk** [comment]
Released as part of [9.5.22](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.22).
